L/libpolymake_julia/common.jl

```diff
@@ -5,11 +5,12 @@ import Pkg: PackageSpec
 import Pkg.Types: VersionSpec
 
 name = "libpolymake_julia"
-version = v"0.3.0"
+version = VersionNumber(0, 4, julia_version.minor)
```

Comment:
Note this quirky version -- we should have something newer than 0.3, so at least 0.4;  but at this point, we are decoupling from the upstream version anyway, so we might as well use 3.0 or 4.0 or whatever else strikes your fancy.

I would also in general like to float the idea of not bothering with on upstream `libpolymake-julia`, and instead use a `GitSource` below. At least I fail to see any benefit from having a version there? It just adds extra work?

Reply:
There is an extra version information in the library headers that is checked at runtime which I like to keep that way as a failsafe and for people using overrides. So I will update that to 0.4 and create a new tag.
Also wo do have full control over the upstream versions here so there is really no need for decoupling the version.